docker-compose.yml

```diff
@@ -96,7 +96,7 @@ services:
             - DASHBOARD_URL=${DASHBOARD_URL:-https://ceph2:11000}
 
     grafana:
-        image: ${GRAFANA_IMAGE:-ceph/ceph-grafana}
+        image: ${GRAFANA_IMAGE:-quay.io/ceph/ceph-grafana:8.3.5}
```

Comment:
@aaSharma14 while testing this I realized that I just had to do a `docker-compose pull grafana` in local to pull the latest grafana image. Because `ceph/ceph-grafana` should pull the latest image in the repo right? And if we pin the grafana version we'll have to change it again once we push a newer version in ceph/ceph-grafana.

I don't think we need to worry about version pinning here since we are the only ones who is going to push to `ceph/ceph-grafana`. 

The other thing is the `quay.io`. if we don't provide it, does the docker-compose pull the image from docker or quay?